README.md

```diff
@@ -110,14 +113,14 @@ options such as building a Swift-compatible LLDB, building the Swift Package
 Manager, building for iOS, running tests after builds, and more. It also
 supports presets, which you can define for common combinations of build options.
 
-View the inline help to find out more, in particular the section "Typical uses"
+View the inline help to find out more, in particular the section "Typical uses":
 
     utils/build-script -h
 
 A basic command to build Swift with optimizations and run basic tests with
 Ninja:
 
-    utils/build-script -r -t
+    utils/build-script --release --test
```

Comment:
Ah, IIRC `-r` is `--release-debuginfo`, not just `--release`. (That's `-R`.) I know you make that distinction below, but for the *very first thing* people build they probably want the debug info and assertions.